main/plots.py

```diff
@@ -111,229 +117,73 @@ def create_capacity_planning_plot(start_date: datetime, end_date: datetime) -> R
         end_date: datetime object representing the end of the plotting period
         x_range: (optional) tuple of datetimes to use as the x_range for the displayed
             plot
+        selected_projects: (optional) list of selected project name(s)
+        selected_users: (optional) list of selected user name(s)
```

Comment:
And update the docstring
```suggestion
        use_projects: (optional) list of selected project name(s)
        use_users: (optional) list of selected user name(s)
```